easy-ui-react/src/FocusedProductLayout/FocusedProductLayout.module.scss

```diff
@@ -0,0 +1,56 @@
+@use "../styles/common" as *;
+@use "../styles/unstyled";
+@use "../ProductLayout/mixins" as ProductLayout;
+
+.FocusedProductLayout {
+  @include component-token("focused-product-layout", "content-order", 1);
+  @include component-token("focused-product-layout", "side-panel-order", 2);
+
+  display: flex;
+  flex-direction: column;
+
+  @include ProductLayout.breakpoint-large-screen {
+    padding: design-token("space.2") design-token("space.4");
+    padding-bottom: 0;
+  }
+}
+
+.innerContainer {
+  display: flex;
+  flex-direction: column;
+  min-height: 100svh;
+  overflow: hidden;
+
+  @include ProductLayout.breakpoint-large-screen {
+    min-height: calc(100svh - #{design-token("space.2")});
```

Comment:
**question** not saying this should be changed or anything, more or a question out of curiosity, but does `svh` guard against anything or give us a win over setting the viewport height in a different way?

Reply:
good question. you probably already know about `svh`, `lvh`, and `dvh` for setting heights based on different mobile viewport configurations: https://web.dev/articles/viewport-units.

i was wanting to make sure that the layout is always at least as tall as the smallest visible mobile viewport, without stretching into the area of the browser below that (under the status bar). if it stretches into that area it causes the page to have a scrollbar. `lvh` would cause it to stretch into that bottom area, and `dvh` causes layout shift issues due to it changing the element size as the visible viewport changes.

does that make sense and/or answer the question?